rh-tictactoe/src/app/screens/Game/index.js

```diff
@@ -20,6 +20,10 @@ class GameContainer extends React.Component {
     this.props.dispatch(actionCreator.jumpTo(value, step));
   };
 
+  logOut = () => {
+    localStorage.removeItem('token');
```

Comment:
this should be in a service, and you should delete any other data too,, like the email and the `Authorization` api header

Reply:
Done, logOut now calls an action and cleans all user data. Take into acount that the logout action will be moved to a topbar redux folder in the next branch.